src/portable/ReactUi/react/src/components/ui-controls/AnchorHelp.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import './AnchorHelp.sass';
+
+interface IProps {
+    id: string;
+    color?: string;
+    onClick: (value: string) => any;
+};
+
+class AnchorHelp extends React.Component<IProps, object> {
+    constructor(props: IProps) {
+        super(props);
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    public handleClick(event: any) {
+        this.setState({ ...this.state, current: event.target.value })
```

Comment:
I am not sure we need to set the state as we don't define the state or use this value anywhere...